http-ui/src/main.rs

```diff
@@ -414,48 +390,46 @@ async fn main() -> anyhow::Result<()> {
                         Ok(_count) => wtxn.commit().map_err(Into::into),
                         Err(e) => Err(e.into())
                     }
-                },
+                }
                 UpdateMeta::Settings(settings) => {
                     // We must use the write transaction of the update here.
                     let mut wtxn = index_cloned.write_txn()?;
                     let mut builder = update_builder.settings(&mut wtxn, &index_cloned);
 
                     // We transpose the settings JSON struct into a real setting update.
-                    if let Some(names) = settings.searchable_attributes {
-                        match names {
-                            Some(names) => builder.set_searchable_fields(names),
-                            None => builder.reset_searchable_fields(),
-                        }
-                    }
+                    match settings.searchable_attributes {
+                        Setting::Set(searchable_attributes) => builder.set_searchable_fields(searchable_attributes),
+                        Setting::Reset => builder.reset_searchable_fields(),
+                        _ => ()
+                    };
```

Comment:
```suggestion
                    }
```

Could you please change that in all other similar places, please?